stable/alpine/Dockerfile

```diff
@@ -1,19 +1,23 @@
-FROM alpine:3.7
+FROM alpine:3.11
 
-LABEL maintainer="NOS Inovação <nosi.metadata@nos.pt>"
+LABEL maintainer="NOS Inovação <nosi.platforms@nos.pt>"
 
-ENV NGINX_VERSION 1.16.1
-ENV NJS_VERSION 0.3.5
+ENV NGINX_VERSION 1.18.0
+ENV NJS_VERSION 0.4.0
 
 # Ensure git
 RUN apk update && apk upgrade \
 	&& apk add --no-cache --virtual .build-deps bash git libmaxminddb-dev autoconf automake
```

Comment:
The libmaxminddb was already in this statement, does it make sense to be here and compile directly from the sourcecode below?

Reply:
I thought I had removed that package...
The nginx build does not find the necessary libs from that package (maybe it's too old?) so I've opted to build it from source as it is quite simple and has no dependencies other than libc.

Reply:
ok, fine by me :)